src/Components/Components/src/ParameterView.cs

```diff
@@ -101,7 +100,7 @@ public TValue GetValueOrDefault<TValue>(string parameterName)
         /// <param name="defaultValue">The default value to return if no such parameter exists in the collection.</param>
         /// <returns>The parameter value if found; otherwise <paramref name="defaultValue"/>.</returns>
         public TValue GetValueOrDefault<TValue>(string parameterName, TValue defaultValue)
-            => TryGetValue<TValue>(parameterName, out TValue result) ? result : defaultValue;
+            => TryGetValue<TValue>(parameterName, out var result) ? result : defaultValue;
```

Comment:
Can you help me understand what this change is doing? Would the user have to cast to TValue themselves?

Reply:
I think I could revert this change. I was looking at what it would take to make `GetValueOrDefault` take a nulllable `defaultValue` but it became very messy.

Reply:
```suggestion
            => TryGetValue<TValue>(parameterName, out TValue result) ? result : defaultValue;
```